maturity-level-one/src/Controllers/PlayersController.cs

```diff
@@ -112,15 +108,15 @@ public async Task<IActionResult> UpdateIncremental(int id, [FromBody] PlayerDto
                 return NotFound();
             }
 
-            var playerToBeUpdated= Mapper.Map<Player>(player);
+            var playerToBeUpdated = Mapper.Map<Player>(player);
             playerToBeUpdated.Id = id;
 
             await _worldCupRepository.ApplyPatch<Player, PlayerDto>(playerToBeUpdated, player);
             return NoContent();
         }
 
         [HttpPatch("{id}/update")]
-        [SwaggerOperation("update-player-jsonpatch")]
+        [SwaggerOperation("Players_Patch")]
```

Comment:
```suggestion
        [SwaggerOperation("Players_UpdateIncrementalJsonPatch")]
```

Reply:
That's good for me as well, as long as it reflects in the method name.